scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -157,11 +157,11 @@ public static BuildTargetResult PackageCoreHost(BuildTargetContext c)
 
             var version = buildVersion.NuGetVersion;
             var content = $@"{c.BuildContext["CommitHash"]}{Environment.NewLine}{version}{Environment.NewLine}";
-            File.WriteAllText(Path.Combine(c.BuildContext.BuildDirectory, "src", "corehost", "packaging", "version.txt"), content);
-            string corehostSrcDir = Path.Combine(c.BuildContext.BuildDirectory, "src", "corehost");
+            var buildToolsDir = Path.Combine(c.BuildContext.BuildDirectory, "packaging", "buildtools");
```

Comment:
actually thinking about that more given that buildtools is miss-leading in this repo we should avoid that.